src/tokens/selectors.ts

```diff
@@ -88,6 +89,22 @@ export const tokensByIdSelector = createSelector(
   }
 )
 
+export const importedTokensInfoSelector = createSelector(
+  (state: RootState) => state.tokens.importedTokens,
+  (manualTokens) => {
+    const tokenBalances: StoredTokenBalances = {}
+    for (const manualToken of Object.values(manualTokens)) {
+      tokenBalances[manualToken.tokenId] = {
+        ...manualToken,
+        // Force imported tokens to be visible even with zero balance.
+        showZeroBalance: true,
+        balance: null,
```

Comment:
does this set the balance to be 0 always, so we never show the balance? do we need this?

Reply:
I'm explicitly setting the balance here to use the same interface `StoredTokenBalances`.
The balances are updated in the saga.
This selector returns only the "static" information for the token.

If we return the balance here, we might return a wrong value (since this would be the balance value when the token was imported).